### ISSUE TITLE
Migrate content from portaljs.org/guide to this site

### ISSUE THREAD

**Issue by rufuspollock** [datopian/makeitmarkdown#7]
Which itself was migrated from markdownbased site - see https://github.com/datopian/portaljs/issues/994

Content to migrate is therefore likely covered in this PR https://github.com/datopian/portaljs/pull/996 i.e. guide page plus some howtos.

## Acceptance

- [x] Migrate content from portaljs **✅2023-12-20 DONE: see next two items:**
  - [x] Identify content **✅2023-12-20 DONE: see detailed list of items in tasks section below**
  - [x] Add the content **✅2023-12-20 DONE: see updates on items list in tasks section below**
- [x] Also check if anything left on markdownbased https://github.com/datopian/markdownbased that should move (and then close that down) **✅2023-12-19 did this in #8**

Finally

- [ ] Remove from portaljs.org (and add redirect?) **❌2023-12-21 WONTFIX. I think we should delete only once Makeitmarkdown is complete, as backup in case anything goes wrong or we need to reference something. Added this to [inbox of outstanding tasks](https://github.com/datopian/makeitmarkdown/issues/20) to revisit later**

## Tasks

- [x] Make list of things to migrate **See tasks below**
- [x] Migrate content **(Done this via Obsidian)**
  - [x] guide index page **See [flowershow](https://makeitmarkdown.flowershow.app/)**
  - [x] guide index page images **(Is there a way to do this that doesn't involve manually downloading each image from GitHub and importing into Obsidian?)**
  - [x] tutorial pages **See [tutorial 1](https://makeitmarkdown.flowershow.app/content/tutorials/create-a-website-from-scratch) and [tutorial 2](https://makeitmarkdown.flowershow.app/content/tutorials/edit-a-website-locally). Tutorials 3 and 4 have not yet been written**
  - [x] tutorial pages images 
  - [x] howtos **See links to 6 howtos at the bottom of the [index page](https://makeitmarkdown.flowershow.app/)**

**Comment by elisapaka** [comment]
@rufuspollock I've transferred over the content of the guide from portaljs to [makeitmarkdown](https://github.com/datopian/makeitmarkdown)/[flowershow](https://makeitmarkdown.flowershow.app/). Please review and let me know if anything needs changing. Some notes:

- There is currently no 'home' or 'next' button on the howto/tutorial pages, but I thought we should think about how we want to structure the site first to avoid unnecessary work.
- In the 'content' folder in makeitmarkdown, I've replaced the 'blog' folder with a 'tutorials' folder. 
- I have not transferred over all the images on the tutorial pages into the new assets folder. They are currently showing on flowershow but linking to the portaljs repo. I didn't know of a way to transfer the contents of the assets folder automatically from one repo to another in a way that they show up in the Obsidian files, and it would be a lot of work to do it manually for each image. Since we might re-do the tutorials and thus not keep these images, I'll hold off from doing this until we've confirmed what we're keeping.

**Comment by elisapaka** [comment]
FIXED. PortalJS content mingrated to [Make It Markdown]([makeitmarkdown.flowershow.app](https://makeitmarkdown.flowershow.app/)) on flowershow app.

@rufuspollock Closing this issue as migration is complete. NB images still link to the portaljs repo, so I don't want to delete anything on there just yet. I haven't migrated the images to the makeitmarkdown repo (except for the ones on the index page) because I imagine I'll redo the screenshots as we'll be using a slightly different method for the tutorials (and migrating them manually would be a lot of unnecessary work). But until I've finalised what I need I don't want to delete anything on portaljs, so I've marked the last acceptance item as WONTFIX. **UPDATE 23-12-29: Added this to [inbox of outstanding tasks](https://github.com/datopian/makeitmarkdown/issues/20) to revisit later**

**Comment by rufuspollock** [comment]
@elisapaka remember when updating here i'd recommend updating the tasks with the resulting outputs as per https://lifeitself.org/tao/issues#completing-tasks (i.e. where you migrated stuff to) - we do this especially here because acceptance is high level.

Note also the acceptance updates are pretty general and don't tell me much. Better to have something like:

- [x] Migrate content from portaljs **✅2023-12-20 DONE: see next two items.**
  - [x] Identify content **✅2023-12-23 see the detailed list of items below in tasks section**
  - [x] Add the content **✅2023-12-23 migrated guide front page to front page LINK and each of the 2 complete tutorials to XXX and YYYY (link). Also migrated all 6 howtos - see links below [OR: see the detailed list of items below and update tasks below]**

Instead of what we have:

- [x] Migrate content from portaljs **✅2023-12-20 DONE: see on [makeitmarkdown](https://github.com/datopian/makeitmarkdown), and on [flowershow](https://makeitmarkdown.flowershow.app/)**
  - [x] Identify content
  - [x] Add the content

**Comment by elisapaka** [comment]
@rufuspollock Great, thank you for the feedback! I've edited the updates as per your suggestion.

**Comment by rufuspollock** [comment]
Great. Last comment: we can easily forget the stuff about removing from portaljs and doing redirect so I would capture that somewhere like a new issue or an inbox for end of this phase of work and mention that in your issue completing comment.

**Comment by elisapaka** [comment]
@rufuspollock Perfect, I've added it here: https://github.com/datopian/makeitmarkdown/issues/20
